Results.js

```diff
@@ -0,0 +1,13 @@
+/* I think that the final version of this page should just be some 
+animation (like a bar filling up or something) which changes
+based on how many times the knight was hit */
+import { StyleSheet, Text, View, Button } from 'react-native';
+
```

Comment:
Example comment:
// Display the final score. Expects route.params.counter to be set.